User/Library/UserMisc.c

```diff
@@ -8,6 +8,8 @@
 #include <Library/UefiApplicationEntryPoint.h>
 #include <Library/DebugLib.h>
 
+BOOLEAN mIsAvxEnabled;
```

Comment:
There is a convention that any global variable declaration that is not STATIC gets GLOBAL_REMOVE_IF_UNREFERENCED.